components/oracle.tsx

```diff
@@ -37,36 +39,52 @@ export default function Oracle({ children }) {
     }
   ]);
 
+  async function loadData() {
+    try {
+      const networkAddress = handleNetworkAddress(activeNetwork);
+      const dao = new DAO({
+        web3Connection: DAOService.web3Connection,
+        skipWindowAssignment: true
+      });
+
+      await dao.loadNetwork(networkAddress);
+
+      Promise.all([
+        DAOService.getClosedBounties(),
+        DAOService.getOpenBounties(),
+        DAOService.getTotalSettlerLocked(),
+        getTotalUsers()
+      ])
+      .then(([closed, inProgress, onNetwork, totalUsers]) => {
+        setInfos([
+          {
+            value: inProgress,
+            label: t("common:heroes.in-progress")
+          },
+          {
+            value: closed,
+            label: t("common:heroes.bounties-closed")
+          },
+          {
+            value: onNetwork,
+            label: t("common:heroes.bounties-in-network"),
+            currency: "BEPRO"
+          },
+          {
+            value: totalUsers,
+            label: t("common:heroes.protocol-members")
+          }
+        ]);
+      });
+    } catch (error) {
+      console.log("Failed to load oracle data", error, activeNetwork);
```

Comment:
This error should be _seen_ as well -- add a toastError here so we are warned without having to look into the console log (which will also help with understanding when something went wrong)